utilities/state_space.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+from wpimath import units
+from wpimath.system import LinearSystem_2_1_1
+from wpimath.system.plant import DCMotor
+
+
+def single_jointed_arm_system(
+    motor: DCMotor, J: units.kilogram_square_meters, gearing: float
+) -> LinearSystem_2_1_1:
+    A = np.array(
+        [[0.0, 1.0], [0.0, -(gearing**2) * motor.Kt / (motor.Kv * motor.R * J)]]
+    )
```

Comment:
nit: let's write this so this looks more like a matrix
```suggestion
    A = np.array([
        [0.0, 1.0],
        [0.0, -(gearing**2) * motor.Kt / (motor.Kv * motor.R * J)],
    ])
```